File: aifupan_project/fupan-server/replay-words/src/main/java/com/jiuyu/replay/words/vo/HistoryParagraphVo.java
package com.jiuyu.replay.words.vo;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.io.Serializable;
import java.util.Date;

/**
 * ai问答的历史分析段落记录信息
 *
 * @author lj
 * @email 
 * @date 2025-03-22 15:36:36
 */
@Data
@Schema(description = "ai问答的历史分析段落记录信息")
public class HistoryParagraphVo implements Serializable {
	private static final long serialVersionUID = 1L;

	/**
	 * id
	 */
	@Schema(description = "id")
	private Long id;
	/**
	 * 租户id
	 */
	@Schema(description = "租户id")
	private Long tenantId;
	/**
	 * 用户id
	 */
	@Schema(description = "用户id")
	private Long userId;
	/**
	 * 别名
	 */
	@Schema(description = "别名")
	private String alias;
	/**
	 * 资产类型 0视频，1文件，2对比分析
	 */
	@Schema(description = "资产类型 0视频，1文件，2对比分析")
	private Integer sourceType;
	/**
	 * 资产id
	 */
	@Schema(description = "资产id")
	private String sourceId;
	/**
	 * 助手类型 0运营助手 1违规助手
	 */
	@Schema(description = "助手类型 0运营助手 1违规助手")
	private Integer type;
	/**
	 * 历史段落的code
	 */
	@Schema(description = "历史段落的code")
	private String code;
	/**
	 * 段落内容
	 */
	@Schema(description = "段落内容")
	private String content;
	/**
	 * 最后修改时间
	 */
	@Schema(description = "最后修改时间")
	private Date updateDate;
	/**
	 * 创建时间
	 */
	@Schema(description = "创建时间")
	private Date createDate;


}

File: aifupan_project/ReviewAnalysis/vo/ai/HistoryParagraphVo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.vo.ai
{
    public class HistoryParagraphVo
    {

        /// <summary>
        /// 助手类型 0运营助手 1违规助手
        /// </summary>
        public int type { get; set; }

        /// <summary>
        /// 历史段落的code 
        /// </summary>
        public string code { get; set; }

        /// <summary>
        /// 别名 
        /// </summary>
        public string alias { get; set; }

        /// <summary>
        /// 段落内容
        /// </summary>
        public string content { get; set; }

        /// <summary>
        /// 来源的id
        /// </summary>
        public string sourceId { get; set; }

        /// <summary>
        /// 数据类型 0视频，1文件，2对比分析
        /// </summary>
        public int sourceType { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        public string createDate { get; set; }

    }
}
